src/AspNet.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -9,18 +9,41 @@ public static class OpenIdConnectConstants {
         public static class Claims {
             public const string Active = "active";
             public const string Address = "address";
+            public const string AtHash = "at_hash";
+            public const string Birthdate = "birthdate";
             public const string ClientId = "client_id";
             public const string Confidential = "confidential";
+            public const string Email = "email";
```

Comment:
As `payload` can be anything, not only JWT, I think it'd better be using our own constants. A bit more consistency, because opaque tokens are default ones now and addressing same values using different constants in different places would be a bit misleading.